README.md

```diff
@@ -126,22 +139,6 @@ Some variables are specific to Docker, and the default values are :
 
 Additionally, if `SKIP_PERMISSIONS_CHECKS` is set to "yes", the entrypoint script will not check or set the permissions of files and directories on startup. Users are strongly advised **against** using this option, and efforts have been made to keep the checks as fast as possible. Nonetheless, it may be suitable for some advanced use cases.
 
-## Advanced configuration
-
-Note that nginx will accept by default images up to 100MB (`client_max_body_size 100M`) and that PHP parameters are overridden according to the [recommendations of the Lychee FAQ](https://lycheeorg.github.io/docs/faq.html#i-cant-upload-large-photos).
-
-You may still want to further customize PHP configuration. The first method is to mount a custom `php.ini` to `/etc/php/8.2/fpm/php.ini` when starting the container. However, this method is kind of brutal as it will override all parameters. It will also need to be remapped whenever an image is released with a new version of PHP.
-
-Instead, we recommend to use the `PHP_VALUE` directive of PHP-FPM to override specific parameters. To do so, you will need to mount a custom `nginx.conf` in your container :
-
-1. Take the [default.conf](./default.conf) file as a base
-2. Find the line starting by `fastcgi_param PHP_VALUE [...]`
-3. Add a new line and set your new parameter
-4. Add or change any other parameters (e.g. `client_max_body_size`)
-5. Mount your new file to `/etc/nginx/nginx.conf`
-
-If you need to add (not change) nginx directives, files mounted in `/etc/nginx/conf.d/` will be included in the `http` context.
-
```

Comment:
I was going to object to this, but I guess these overrides aren't really needed any more (unless someone really knows what they're doing, and so can figure it out easily enough).

Reply:
I thought about it, the main reason I am dropping it is because Lychee now supports upload by chunks, as a result the limit on post request is no longer an issue for large files. :)